src/components/Modal/index.tsx

```diff
@@ -74,14 +62,15 @@ class BaseModal extends React.Component<ThemedModalProps, any> {
 		if (!e.defaultPrevented && e.which === 13) {
 			e.preventDefault();
 			e.stopPropagation();
-
-			if (this.props.primaryButtonProps?.disabled) {
+			const firstPrimaryAction = this.props.actions?.find(
```

Comment:
Make sure this works when the primary action is dangerous and is to the left ( so I guess the primary will be to cancel)

Reply:
I did it like this, since I would expect that a button without type or type `none`can just be a `cancel`action which is neither primary nor dangerous. But you clearly made me think that the name is wrong :).

Reply:
I did it like this since I was expecting to not see a button with both `primary` and `dangerous` types. Is this thought wrong ?

Reply:
No you are right, as long as it works fine I'm ok, I just wanted to double-check :)